src/containers/nav-bar.html

```diff
@@ -37,24 +49,32 @@
 
 <aside class="mdc-drawer mdc-drawer--modal">
   <div class="mdc-drawer__content">
-    <nav>
-      <button class="mdc-button nav-button" tabindex="0" data-navigate-to="home">
+    <nav class="mdc-list">
+      <button class="mdc-list-item mdc-button nav-button" tabindex="0" data-navigate-to="home">
+        <span class="material-icons">home</span>
```

Comment:
The icons should follow the mdc examples:
`<i class="material-icons mdc-list-item__graphic" aria-hidden="true">inbox</i>`

- Should use the `<i>` element
- Needs the mdc-list-item-graphic class for the spacing (then you shouldn't need the margin you added)
- Needs the aria-hidden attribute for accessibility